supabase/functions/process-transcripts/index.ts

```diff
@@ -0,0 +1,160 @@
+import { serve } from 'https://deno.land/std@0.170.0/http/server.ts';
+import OpenAI from 'https://deno.land/x/openai@v4.26.0/mod.ts';
+import { corsHeaders } from '../common/cors.ts';
+import { supabaseClient } from '../common/supabaseClient.ts';
+
+const processTranscripts = async (req: Request) => {
+    try {
+        const supabase = supabaseClient(req);
+        const openaiClient = new OpenAI({
+            apiKey: Deno.env.get('OPENAI_API_KEY')!,
+        });
+
+        const { data, error } = await supabase
+            .from('records')
+            .select('raw_text')
+            .eq('processed', false)
+            .order('created_at', { ascending: true })
+            .limit(5);
```

Comment:
More of an exploratory question, but I wonder if we could benefit from putting values like this in a json config env var. That way you don't have to redeploy the edge function just to update one param.